ts/components/screens/OperationResultScreenContent.tsx

```diff
@@ -73,9 +75,14 @@ const OperationResultScreenContent = ({
   action,
   secondaryAction,
   children,
-  testID
+  testID,
+  isVisibleHeader
```

Comment:
```suggestion
  isHeaderVisible
```

`isVisibleHeader`, while understandable, inverts the typical order of descriptors and subjects. The preferred choice would be `isHeaderVisible`. This naming convention follows a common pattern in programming where the "is" prefix indicates a boolean value, followed by the subject (`Header`) and the property being described (`Visible`). What do you think?

Reply:
Yes! Sure! Thanks for the suggestion!

Reply:
done!